src/pages/data-storage/collections.md

```diff
@@ -18,13 +18,13 @@ particular. The following topics are covered:
 
 ## Understand how collections work
 
-One of Blockstack's goals is to give users true data ownership by enabling _data portability_. Data portability allows
+One of the goals of the Stacks ecosystem is to give users true data ownership by enabling _data portability_. Data portability allows
 users to login with their digital ID on any app and have access to the same data. For example, if a user adds a photo of
 a Hawaiian vacation in one app, that photo enters the user's data pool. Then, when the user opens a second app, that
 same photo is available to the second app because the user data, including the photo, is shared via the user's
 decentralized ID.
 
-How do collections work? Blockstack builds a library containing commonly used data schemes. Developers use these classes
+How do collections work? Hiro PBC builds a library containing commonly used data schemes. Developers use these classes
```

Comment:
Can we use Stacks here to remain consistent on the language 🙏

Reply:
Updated!